evap/staff/views.py

```diff
@@ -1040,6 +1040,41 @@ def course_copy(request, course_id):
     )
 
 
+@require_POST
+@manager_required
+def create_exam_evaluation(request):
+    evaluation = get_object_from_dict_pk_entry_or_logged_40x(Evaluation, request.POST, "evaluation_id")
+    exam_date = datetime.today()  # request.POST.get("date")
+    if evaluation.is_single_result:
+        raise SuspiciousOperation("Creating an exam evaluation for a single result evaluation is not allowed")
+
+    if evaluation.course.evaluations.filter(name_de="Klausur", name_en="Exam").exists():
+        messages.error(request, _("An exam evaluation already exists for this course."))
+        return HttpResponse()  # 200 OK
+
+    evaluation.weight = 9
+
+    course_evaluation_end_date = exam_date - timedelta(days=1)
+    if evaluation.vote_start_datetime > course_evaluation_end_date:
+        raise SuspiciousOperation("The selected date is before the start date")
+    evaluation.vote_end_date = course_evaluation_end_date
+
+    exam_evaluation = Evaluation(
+        course=evaluation.course, name_de="Klausur", name_en="Exam", weight=1, is_rewarded=False
+    )
+    exam_evaluation.vote_start_datetime = datetime.combine(exam_date + timedelta(days=1), time(8, 0))
+    exam_evaluation.vote_end_date = exam_date + timedelta(days=3)
+    exam_evaluation.save()
+    exam_evaluation.participants.set(evaluation.participants.all())
+    for contribution in evaluation.contributions.all().filter(contributor__isnull=False):
+        exam_evaluation.contributions.create(contributor=contribution.contributor)
+    exam_evaluation.general_contribution.questionnaires.set([Questionnaire.objects.get(id=1)])
```

Comment:
We can't assume that the questionnaire with `id=1` is the correct questionnaire, it may not even be valid. Please read the specification in the issue:
```
All questionnaires listed in a new settings variable EXAM_QUESTIONNAIRES (initially set to contain the id of the "Miscellaneous" questionnaire) will be added to the exam evaluation as general questionnaires.
```